spec/setup_acceptance_node.pp

```diff
@@ -8,3 +8,12 @@
 package { 'dhcping':
   ensure => installed,
 }
+
+if $facts['os']['family'] == 'RedHat' {
+  if Integer($facts['os']['release']['major']) >= 8 {
```

Comment:
It's a good practice not to think about releases as integers. We've taking that shortcut in tests, but be aware that we normally use `versioncmp()` when dealing with this. In Ubuntu `major` is `18.04`. In this case it's safe, but just wanted to give you a heads up.

I also think this isn't that important: we don't do it in other modules and tests pass without it so I'm leaning to keeping it out. Only include what's strictly needed. Perhaps we need to set the locale to `C.UTF-8` instead to avoid it.

Reply:
As I understand it, the package would in fact be present on any standard EL8 install, it's just missing from the very minimal EL8 container image used for testing. For that reason, I'm not seeing a downside to keeping it, since it not only suppresses the very annoying warnings that clutter the test output but also brings the Travis test environment incrementally closer to a real world use case.

@ekohl is this just a general matter of principle that you prefer to omit anything from the test environment that isn't strictly necessary for tests to pass?

Reply:
> brings the Travis test environment incrementally closer to a real world use case.

I disagree with this. In the US the EN locales might be considered core, but outside of it a user might only install their local ones. Maybe this is uncommon in practice, but the container image is minimal for a reason.

> is this just a general matter of principle that you prefer to omit anything from the test environment that isn't strictly necessary for tests to pass

Yes. I'm a fan of being minimalistic because things might end up depending on it.

I would be interested in knowing if running `LANG=C.UTF-8 rake beaker` in the EL8 also removes the warnings.

Reply:
So I thought that `glibc-langpack*` packages come pre-installed for multiple languages but I double checked on an EL8 system and it turns out that is not the case.

I will try setting the environment variable in `.travis.yml` to see if it stops the constant warnings.